src/bomberman/utils/Animator.java

```diff
@@ -0,0 +1,37 @@
+package bomberman.utils;
+
+import javafx.scene.canvas.GraphicsContext;
+import javafx.scene.image.Image;
+
+public class Animator
+{
+	/**
+	 *
+	 * @author CoreyHendrey
+	 */
+   static Image img;
+
+    public static void init() {
+        img = ImageUtils.loadImage("src/Resources/img/sprites.png");
+    }
+
+    // "ActualSize" refers to how many pixels across to look for the next sprite
+
+    public static void playAnimation(GraphicsContext gc, double time, Sprite sprite) {
+    	playAnimation(gc, time, sprite.actualSize, sprite.spriteLocationOnSheetX, sprite.spriteLocationOnSheetY, sprite.numberOfFrames, sprite.x, sprite.y, sprite.width, sprite.height, sprite.scale, sprite.leftToRight);
+    }
+
+    public static void playAnimation(GraphicsContext gc, double time, int actualSize, int startingPointX, int startingPointY, int numberOfFrames, int x, int y, int width, int height, boolean leftToRight) {
+    	playAnimation(gc, time, actualSize, startingPointX, startingPointY, numberOfFrames, x, y, width, actualSize, 1, leftToRight);
+    }
+
+    public static void playAnimation(GraphicsContext gc, double time, int actualSize, int startingPointX, int startingPointY, int numberOfFrames, int x, int y, int width, int height, double scale, boolean leftToRight) {
```

Comment:
what is the use of index here
Please explain how this method works.

Reply:
It receives the current time from the gameloop and then converts it into an index for which sprite to display based on the framespeed you’ve chosen.  Index 0 will show the first sprite.

The method just samples the Sprites.png based on the arguments you send it to display the correct animations easily. Look in PlayerEntity at the Sprites being created to get an idea of how it works.